Client/Frontend/Browser/Playlist/Cache/PlaylistCacheLoader.swift

```diff
@@ -0,0 +1,466 @@
+        "snd",
+        "ac3",
+        "eac3",
+        "flac",
+        "aac",
+        "mp2",
+        "pls",
+        "avi",
+        "webm",
+        "ogg",
+        "mpg",
+        "mpg4",
+        "mpeg",
+        "mpg3",
+        "wma",
+        "wmv",
+        "swf",
+        "flv",
+        "mng",
+        "asx",
+        "asf",
+        "mkv"
+    ]
+    
+    private let mimeTypeMap = [
+        "audio/x-wav": "wav",
+        "audio/vnd.wave": "wav",
+        "audio/aacp": "aacp",
+        "audio/mpeg3": "mp3",
+        "audio/mp3": "mp3",
+        "audio/x-caf": "caf",
+        "audio/mpeg": "mp3", //mpg3
+        "audio/x-mpeg3": "mp3",
+        "audio/wav": "wav",
+        "audio/flac": "flac",
+        "audio/x-flac": "flac",
+        "audio/mp4": "mp4",
+        "audio/x-mpg": "mp3", //maybe mpg3
+        "audio/scpls": "pls",
+        "audio/x-aiff": "aiff",
+        "audio/usac": "eac3",  // Extended AC3
+        "audio/x-mpeg": "mp3",
+        "audio/wave": "wav",
+        "audio/x-m4r": "m4r",
+        "audio/x-mp3": "mp3",
+        "audio/amr": "amr",
+        "audio/aiff": "aiff",
+        "audio/3gpp2": "3gp2",
+        "audio/aac": "aac",
+        "audio/mpg": "mp3", //mpg3
+        "audio/mpegurl": "mpg", // actually .m3u8, .m3u HLS stream
+        "audio/x-m4b": "m4b",
+        "audio/x-m4p": "m4p",
+        "audio/x-scpls": "pls",
+        "audio/x-mpegurl": "mpg", // actually .m3u8, .m3u HLS stream
+        "audio/x-aac": "aac",
+        "audio/3gpp": "3gp",
+        "audio/basic": "au",
+        "audio/au": "au",
+        "audio/snd": "snd",
+        "audio/x-m4a": "m4a",
+        "audio/x-realaudio": "ra",
+        "video/3gpp2": "3gp2",
+        "video/quicktime": "mov",
+        "video/mp4": "mp4",
+        "video/mp4v": "mp4",
+        "video/mpg": "mpg",
+        "video/mpeg": "mpeg",
+        "video/x-mpg": "mpg",
+        "video/x-mpeg": "mpeg",
+        "video/avi": "avi",
+        "video/x-m4v": "m4v",
+        "video/mp2t": "ts",
+        "application/vnd.apple.mpegurl": "mpg", // actually .m3u8, .m3u HLS stream
+        "video/3gpp": "3gp",
+        "text/vtt": "vtt",  // Subtitles format
+        "application/mp4": "mp4",
+        "application/x-mpegurl": "mpg", // actually .m3u8, .m3u HLS stream
+        "video/webm": "webm",
+        "application/ogg": "ogg",
+        "video/msvideo": "avi",
+        "video/x-msvideo": "avi",
+        "video/x-ms-wmv": "wmv",
+        "video/x-ms-wma": "wma",
+        "application/x-shockwave-flash": "swf",
+        "video/x-flv": "flv",
+        "video/x-mng": "mng",
+        "video/x-ms-asx": "asx",
+        "video/x-ms-asf": "asf",
+        "video/matroska": "mkv"
+    ]
+}
+
+class PlaylistWebLoader: UIView, WKNavigationDelegate {
+    private let tab = Tab(configuration: WKWebViewConfiguration().then {
+        $0.processPool = WKProcessPool()
+        $0.preferences = WKPreferences()
+        $0.preferences.javaScriptCanOpenWindowsAutomatically = false
```

Comment:
What do you think about setting `javaScriptEnabled` to false in the playlist web-loader?

Reply:
Can't because I need `javaScriptEnabled` to run my injected scripts and for the pages to load their videos.